tools/generate-helpers/pg-table-bindings/store_test.go.tpl

```diff
@@ -188,13 +188,15 @@ func (s *{{$namePrefix}}StoreSuite) getTestData(access storage.Access) (*{{.Type
 	testCases := map[string]testCase{
 		withAllAccess: {
 			context:                sac.WithAllAccess(context.Background()),
+			expectedIDs:            []string{ {{ "objA" | .Obj.GetID }}, {{ "objB" | .Obj.GetID }} },
```

Comment:
```suggestion
			expectedObjIDs:            []string{ {{ "objA" | .Obj.GetID }}, {{ "objB" | .Obj.GetID }} },
```

`IDs` and `Identifiers` are the same thing, we should specify at least of them to distinguish between them.